packages/orchestration/test/supports.ts

```diff
@@ -103,9 +106,17 @@ export const commonSetup = async (t: ExecutionContext<any>) => {
   });
   const timer = buildZoeManualTimer(t.log);
   const marshaller = makeFakeBoard().getReadonlyMarshaller();
-  const storage = makeFakeStorageKit('mockChainStorageRoot', {
-    sequence: false,
-  });
+  const makeChainStorageNode = prepareChainStorageNode(
+    rootZone.subZone('storage'),
+  );
+  const storage = makeFakeStorageKit(
+    'mockChainStorageRoot',
+    {
+      sequence: false,
+    },
+    rootZone,
+  );
+  storage.rootNode;
```

Comment:
What's going on with this property access?

Reply:
stray from debugging